source/common/sagemath/library.sage

```diff
@@ -35,8 +35,8 @@ class TBIL:
         p+=arrow((0,0),end_coordinate) #TODO hide arrowheads when show_unit_circle
         p+=arc((0,0),0.1,sector=(reference_angle, end_angle),color="black") #TODO add arrowhead
         if show_angle_value:
-            if type(show_angle_value) is string:
-                angle_value_label = show_angle_value:
+            if type(show_angle_value) is str:
+                angle_value_label = show_angle_value
```

Comment:
This is fixing the stupid error from Tuesday on the main branch, instead of just in Abby's branch.